calendarsubscription/http.go

```diff
@@ -43,8 +43,11 @@ func (s *Store) ServeICalData(w http.ResponseWriter, req *http.Request) {
 	}
 
 	_ = filtered
```

Comment:
Good catch! `_` is used whenever there is a value that is not being used. This was there with the previous PR where we were stubbing out the call to the endpoint. But since that is no longer the case, we no longer need this line, since we will be passing `filtered` as an argument to the call.

Reply:
It was there in my PR because filtered wasn't being used, but should be removed now

Reply:
It's a compile error to have an unused var, so `_ = thing` is the convention to work around it for example code.